plugins/azure/sqlserver/auditStorageAuthType.js

```diff
@@ -0,0 +1,72 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Auditing Storage Authentication Type',
+    category: 'SQL Server',
+    domain: 'Databases',
+    description: 'Ensures that managed identity is configured as authentication type when storage account is chosen as the destination for audit logs on SQL server.',
+    more_info: 'Enabling managed identity as authentication type enhances security when using a storage account as the destination for audit logs. Managed Identity can be a system-assigned managed identity or user-assigned managed identity.',
+    recommended_action: 'Ensure managed identity is configured as authentication type when choosing a storage account as the destination for audit logs on SQL server.',
```

Comment:
```suggestion
    recommended_action: 'Ensure managed identity is configured as authentication type when choosing a storage account as the destination for audit logs on SQL server.',
```
maybe we should check for managed identity as authentication type for audit logs regardless of the destination

Reply:
storage account is the only destination type where managed identity or storage access keys can be configured. any other destination doesn't require any authentication type so.